indra/newview/llviewertexture.cpp

```diff
@@ -511,33 +511,20 @@ void LLViewerTexture::updateClass()
 
     F32 over_pct = (used - target) / target;
 
-    bool is_low = over_pct > 0.f;
+    bool is_sys_low = isSystemMemoryLow();
+    bool is_low = is_sys_low || over_pct > 0.f;
 
     if (isSystemMemoryLow())
```

Comment:
Redundant with `bool is_sys_low = isSystemMemoryLow();` above?

Reply:
Whoops... I had changes that were using "is_sys_low" to make it not slam update all the virtual sizes at once unless we were low on system memory, guess I fat fingered "undo" at some point.  Good catch.